ssn/ssn_separated/ssn.ttl

```diff
@@ -721,7 +617,7 @@ ssn:OperatingRange rdf:type owl:Class ;
                                   http://www.bipm.org/utils/common/documents/jcgm/JCGM_200_2008.pdf
 
 The difference is that here we also allow for qualities that aren't VIM influence quantities [VIM 2.52] - for example, a quantity that alters the power requirements, but doesn't affect the measurement properties - conditions specified in MeasurementCapability should be influence quantities.""" ;
-                   rdfs:comment "The environmental conditions and characteristics of a system or sensor's normal operating environment.  Can be used to specify for example the standard environmental conditions in which the sensor is expected to operate (a Condition with no OperatingProperty), or how the environmental and other operating properties relate: i.e., that the maintenance schedule or power requirements differ according to the conditions." ;
+                   rdfs:comment "The environmental conditions and characteristics of a System's, Sensor's or Actuator's normal operating environment.  Can be used to specify for example the standard environmental conditions in which the Sensor is expected to operate (a Condition with no OperatingProperty), or how the environmental and other operating properties relate: i.e., that the maintenance schedule or power requirements differ according to the conditions." ;
```

Comment:
System's, Sensor's or Actuator's --> System's

(sensors and actuators are systems)